### PR TITLE
DOC: Update installation documentation with new changes

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: doc/source/guide/introduction/installing.rst [hunk 1]
```diff
@@ -22,19 +22,20 @@ If you are a Python novice using Windows, we recommend that you install a full p
...
 ODL fully supports both Python 2 and Python 3, so if you already have a Python distribution (as is the case on most linux distributions) it may be a good idea to keep using it. If you are making a new installation we recommend using Python 3, since Python 2 support will be discontinued in 2020.
 
-Development enviroment
+Development environment
 ----------------------
 Since ODL is object oriented, using an Integrated Development Environment (IDE) is recommended, but not required. The most popular ones are `Spyder
 <https://pythonhosted.org/spyder/>`_ which works on all major platforms and can be installed through ``pip``, and `PyCharm
-<https://www.jetbrains.com/pycharm/>`_ which can be integrated with any text editor of your choice, such as EMACS or Vim.
+<https://www.jetbrains.com/pycharm/>`_ which can be integrated with any text editor of your choice, such as EMACS or Vim. On windows, `Python Tools for Visual Studio
+<https://www.visualstudio.com/vs/python/>`_ is also a valid option, especially for users who are used to Visual Studio.
```
Comment (kohr-h) [inline doc/source/guide/introduction/installing.rst]: are also

File: doc/source/guide/introduction/installing.rst [hunk 2]
```diff
@@ -152,33 +155,39 @@ Unit tests in ODL are based on pytest_, and coverage reports are created by the
...
 
+
 Compiled extensions
 ===================
 There are several extensions to ODL that require you to compile external code, these include
 
 CUDA backend for linear arrays
 ------------------------------
-If you also wish to use the (optional) CUDA extensions you need to run
+If you also wish to use the (optional) CUDA extensions you need to run (in a directory of your choosing)
```
Comment (kohr-h) [inline doc/source/guide/introduction/installing.rst]: choice

File: doc/source/guide/introduction/installing.rst (hunk 1 above)
Comment (adler-j) [inline doc/source/guide/introduction/installing.rst]: Well it is "a set of tools", so I'd singular. Thats how they do it themselves, e.g.

> Python Tools for Visual Studio is a completely free extension, developed and supported by Microsoft with contributions from the community.

**Comment by adler-j** [comment]
Merge after CI

**Pull request merged**